tests/src/data/use-cases/db-forget-password.spec.ts

```diff
@@ -136,7 +138,7 @@ describe('DbForgetPassword UseCase', () => {
     it('Should return false if userId does not exist', async () => {
       const { sut, guardianRepositoryStub } = makeSut()
       jest.spyOn(guardianRepositoryStub, 'updateVerificationToken').mockResolvedValue(false)
-      jest.spyOn(guardianRepositoryStub, 'loadByEmail').mockResolvedValue(undefined)
+      jest.spyOn(guardianRepositoryStub, 'loadByEmail').mockResolvedValue(null)
       const result = await sut.forgetPassword(params)
       expect(result).toBeFalsy()
     })
```

Comment:
This `updateVerificationToken` return is not being used in the production file. The test is passing because of the `loadByEmail` mock return in the test. In addition, the test should expect a false `.toBe(false)`.